web/src/components/overlay/detail/SearchDetailDialog.tsx

```diff
@@ -616,6 +688,35 @@ function ObjectDetailsTab({
               Save
             </Button>
           )}
+          <Dialog
```

Comment:
We should be able to use the existing TextEnterDialog for this

Reply:
Are you thinking of another project? I don't think we have that.

Reply:
Ah, found it `TextEntryDialog`

Reply:
With my original code, I could save an empty value as a sub label. But with `TextEntryDialog` that is prevented. I can add an optional `allowEmpty` prop unless you have any other suggestions.

Reply:
That seems fine